app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt

```diff
@@ -0,0 +1,90 @@
+package com.mapbox.maps.testapp.examples.globe
+
+import android.graphics.Color.rgb
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.MapView
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.Style
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.extension.style.atmosphere.generated.atmosphere
+import com.mapbox.maps.extension.style.layers.properties.generated.ProjectionName
+import com.mapbox.maps.extension.style.projection.generated.projection
+import com.mapbox.maps.extension.style.sources.generated.rasterDemSource
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.extension.style.terrain.generated.terrain
+import com.mapbox.maps.plugin.animation.MapAnimationOptions.Companion.mapAnimationOptions
+import com.mapbox.maps.plugin.animation.flyTo
+import com.mapbox.maps.plugin.gestures.OnMapClickListener
+import com.mapbox.maps.plugin.gestures.addOnMapClickListener
+import com.mapbox.maps.testapp.R
+
+/**
+ * Use [MapboxMap.flyTo] on a map with globe projection to slowly zoom to a location.
+ */
+class GlobeFlyToActivity : AppCompatActivity(), OnMapClickListener {
+
+  private lateinit var mapboxMap: MapboxMap
+  private var isAtStart = true
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    val mapView = MapView(this)
+    setContentView(mapView)
+    mapboxMap = mapView.getMapboxMap()
+    mapboxMap.loadStyle(
+      style(Style.SATELLITE_STREETS) {
+        +projection(ProjectionName.GLOBE)
+        +atmosphere {
+          color(rgb(220, 159, 159)) // Pink fog / lower atmosphere
+          highColor(rgb(220, 159, 159)) // Blue sky / upper atmosphere
+          horizonBlend(0.4) // Exaggerate atmosphere (default is .1)
+        }
+        +rasterDemSource("raster-dem") {
+          url("mapbox://mapbox.terrain-rgb")
+        }
+        +terrain("raster-dem") {
+          // camera seems to be a bit jumping on high zoom level - check with gl-native
+          exaggeration(1.5)
```

Comment:
nit: extract to constants.

Reply:
I'm keeping the code as close to gl-js as possible so will leave as is here 😉